### PR TITLE
adds empty state angular code and points to the correct chart for ang…

### PR DESCRIPTION
**Pull request by LHinson**
- adds empty state angular code

- points to the correct chart for angular area chart

### CODE REVIEW

**Comment by jgiardino** [comment]
The path for the area chart displays an example of the angular version (which is different from the html example, but this is fine).

The path for the empty state example is incorrect. It's missing 'patternfly-org' and should be as shown below:

/patternfly-org/components/angular-patternfly/dist/docs/partials/api/patternfly.views.component.pfEmptyState.html

However, when I view this locally with the correct path, the Empty State does not display as expected. Other contents of the Angular code tab display, like the parameters and sample code, but the actual example does not render at all.

**Comment by LHinson** [comment]
@dabeng could you help look at what's going on with the angular example here?

**Comment by dabeng** [comment]
Hi @jgiardino , could you tell me how did you review the code tab of angular empty state ?

**Comment by LHinson** [comment]
@dabeng Jenn is on PTO

She did a build of the site though (using my PR) to see that the angular version did not look as expected

**Comment by dabeng** [comment]
Hi @jeff-phillips-18 , I need your help.

1. under the directory _build of repo patternfly-org, I set new value to variable code_angular in the file "pattern-library/communication/empty-state/index.md" as shown below:
```html
---
layout: page-pattern
...
code_angular: /components/angular-patternfly/dist/docs/partials/api/patternfly.views.component.pfEmptyState.html
...
---
```

2. under the _build directory, run command "bundle exec jekyll serve"
3. I got the following snapshot:
screen shot 2017-09-01 at 2 55 08 pm

It seems that everything is OK except for Demo. I think maybe empty-state page missed some js references.

**Comment by dabeng** [comment]
Hi @jgiardino , I guess that you got the following result when you built locally.
untitled

right?

**Comment by jeff-phillips-18** [comment]
Looks like you now need to add the dndLists module

<script src="/components/angular-patternfly/dist/docs/grunt-scripts/angular-drag-and-drop-lists.js"></script>

I'm not sure where this is done in this repo, @bleathem ?

**Comment by bleathem** [comment]
You can specify additional js files to include via the yaml frontmatter
eg.:
```
:url-js-extra: ['components/matchHeight/jquery.matchHeight-min.js','components/angular-patternfly/dist/docs/grunt-scripts/angular-drag-and-drop-lists.js']
```

See for example https://github.com/patternfly/patternfly-design/blob/master/pattern-library/content-views/card-view/site.md

**Comment by jgiardino** [comment]
Hi @dabeng, Yes, that's the same result I got. It probably would have helped if I had taken a screen capture at the time that I saw it :-)

**Comment by LHinson** [comment]
Thanks @dabeng

File: pattern-library/communication/empty-state/site.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: page-pattern
 overview: pattern-library/communication/empty-state/design/overview.md
 design: pattern-library/communication/empty-state/design/design.md
 code_html: code/communication/empty-state/code.md
-code_angular: false
+code_angular: /components/angular-patternfly/dist/docs/partials/api/patternfly.views.component.pfEmptyState.html
+url-js-extra: components/angular-patternfly/dist/docs/grunt-scripts/angular-drag-and-drop-lists.js
```
Comment (bleathem) [inline pattern-library/communication/empty-state/site.md]: This line needs to be an array.  I've filed an issue #416

**Pull request merged**